src/tests/class_tests/openms/source/MRMDecoy_test.cpp

```diff
@@ -377,15 +439,14 @@ START_SECTION(OpenMS::TargetedExperiment::Peptide pseudoreversePeptide(OpenMS::T
   OpenMS::TargetedExperiment::Peptide pseudoreverseAASequence_expected;
   pseudoreverseAASequence_expected.sequence = "DITPEPTSETE";
   OpenMS::TargetedExperiment::Peptide pseudoreverseAASequence_result;
-  pseudoreverseAASequence_result = gen.pseudoreversePeptide(pseudoreverseAASequence_target_sequence);
+  pseudoreverseAASequence_result = gen.pseudoreversePeptide_helper(pseudoreverseAASequence_target_sequence);
   TEST_EQUAL(pseudoreverseAASequence_result.sequence, pseudoreverseAASequence_expected.sequence)
 }
-
 END_SECTION
 
 START_SECTION(OpenMS::TargetedExperiment::Peptide reversePeptide(OpenMS::TargetedExperiment::Peptide peptide))
```

Comment:
technically, these are `[EXTRA]` now (since protected), but I'm not sure what the status of the checker is anyway.
Maybe discuss at retreat (will add it to the list).